src/styles/bannerCommon.ts

```diff
@@ -51,6 +51,20 @@ export const styles = {
         }
     `,
 
+    topBar: css`
```

Comment:
This file can be deleted once the other two banners are moved over to the new components